datahub-web-react/src/app/ingest/source/builder/RecipeForm/FormField.tsx

```diff
@@ -74,7 +75,7 @@ function ListField({ field, removeMargin }: ListFieldProps) {
                     {fields.map((item) => (
                         <Form.Item key={item.fieldKey} style={{ marginBottom: '10px' }}>
                             <Form.Item {...item} noStyle>
-                                <Input style={{ width: '80%' }} />
+                                <Input style={{ width: '80%' }} placeholder={field?.placeholder} />
```

Comment:
I don't think you need the `?` after `field` right? since `field` is not optional

Reply:
Removed!